tox.ini

```diff
@@ -92,6 +92,10 @@ extras =
     test
     all: all
 
+# Enable pip to install pre-releases in the dev environment
+pip_pre =
+    dev: True
```

Comment:
What is the difference between these lines of code or typing the following?
```
[testenv:dev]
pip_pre = True
```

Reply:
Your suggestion doesn't work. It only applies to an environment with the exact name `dev`